src/Aspire.Hosting.Azure/AzureCosmosDBEmulatorResource.cs

```diff
@@ -14,7 +14,8 @@ public class AzureCosmosDBEmulatorResource(AzureCosmosDBResource innerResource)
 {
     private readonly AzureCosmosDBResource _innerResource = innerResource;
 
-    public new string Name => _innerResource.Name;
```

Comment:
Assuming this is not breaking as we are just using the base method?

Reply:
This was added last week, during preview4.